### ISSUE TITLE
Question about the input of attribute

### ISSUE THREAD

**Issue by pipi9baby** [edgarschnfld/CADA-VAE-PyTorch#14]
I found that you use ['att'](https://github.com/edgarschnfld/CADA-VAE-PyTorch/blob/master/model/data_loader.py#L91) to be the input of E1 not 'original_att'.
what the difference between 'att' and 'original_att'?

**Comment by edgarschnfld** [comment]
Each class attribute vector is normalized to have the unit l2 norm. So att is the normalized version of original_att.

**Comment by edgarschnfld** [comment]
Further clarification: 
The line
```
self.aux_data = torch.from_numpy(matcontent['att'].T).float().to(self.device)
```
would be equivalent to:
```
self.aux_data = torch.from_numpy(matcontent['original_att'].T).float().to(self.device)
self.aux_data = torch.nn.functional.normalize(self.aux_data, p=2, dim=1)
```
